src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalDayPrice = 40;
+  const mediumDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return days * rentalDayPrice;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * rentalDayPrice) - mediumDiscount;
```

Comment:
You are doing the same calculations `(days * rentalDayPrice)` in every if check. So I suggest putting it in a separate variable